lang/pointer/to.go

```diff
@@ -0,0 +1,33 @@
+package pointer
+
+func ToBool(input bool) *bool {
```

Comment:
and same here - this reads as "pointer . to bool ()" vs it being "pointer . from bool ()"